frontend/public/components/nav/NavHeader.tsx

```diff
@@ -210,12 +238,13 @@ const NavHeader: React.FC<NavHeaderProps> = ({ onPerspectiveSelected }) => {
           data-quickstart-id="qs-perspective-switcher"
         >
           <Dropdown
+            className={cx({ 'oc-nav-header__dropdown-toggle': perspectiveItems.length === 1 })}
```

Comment:
Looks like the cursor is added in the DropdownToggle below.

If you want to add a style there try to describe also the state as we try to follow the https://getbem.com/ conventions

```tsx
className={cx({ 'oc-nav-header__dropdown-toggle--is-empty': perspectiveItems.length === 1 })}
```